config/imports.php

```diff
@@ -51,5 +51,19 @@
             "expiry date",
             "expired?"
         ]
+    ],
+
+    'results' => [
+        'filename_template' => 'import-results-:datetime.csv',
```

Comment:
```suggestion
        'filename_template' => 'results-:id-:datetime.csv',
```

According to T325177 the filename should be a bit more explicit, like so.